build.gradle

```diff
@@ -19,7 +19,8 @@ buildscript {
     ext.experimentalsInTestEnabled = ["-progressive", "-Xuse-experimental=kotlin.Experimental",
                                       "-Xuse-experimental=kotlin.ExperimentalMultiplatform",
                                       "-Xuse-experimental=kotlinx.serialization.ImplicitReflectionSerializer",
-                                      "-Xuse-experimental=kotlinx.serialization.UnstableDefault"
+                                      "-Xuse-experimental=kotlinx.serialization.UnstableDefault",
+                                      "-Xuse-experimental=kotlinx.serialization.InternalSerializationApi"
```

Comment:
I think this particular annotation can be added to main source sets (`experimentalsEnabled`) too. We can use our internal API without limitation)